ui/app/src/components/PreviewAddressBar/PreviewAddressBar.tsx

```diff
@@ -250,9 +251,14 @@ export function PreviewAddressBar(props: AddressBarProps) {
           />
         )}
         <Tooltip title={Boolean(item) ? <FormattedMessage defaultMessage="Options (a)" /> : ''}>
-          <IconButton onClick={onOptions} disabled={!item} size="medium" id="previewAddressBarActionsMenuButton">
-            <MoreRounded sx={alertLevel === 2 ? { visibility: 'hidden' } : undefined} />
-            {!item && (
+          <IconButton
+            onClick={onOptions}
+            disabled={!item || Boolean(error)}
+            size="medium"
+            id="previewAddressBarActionsMenuButton"
+          >
+            <MoreRounded sx={alertLevel === 2 && !error ? { visibility: 'hidden' } : undefined} />
```

Comment:
Let's hide the whole button when in the error state.

Also, notice the tooltip title condition should have included the same disabled condition as the IconButton.